packages/eslint-plugin-ui5-define-jsdocs/tests/lib/rules/duplicate-jsdoc-params.js

```diff
@@ -0,0 +1,83 @@
+"use strict";
+
+const messages = require("../../../lib/messages/manager");
+
+var rule = require("../../../lib/rules/duplicate-jsdoc-params"),
```

Comment:
need eslint to catch these vars -> const/let

Reply:
Done